test/regressions/site/src/tests/Icon/AccentIcon.js

```diff
@@ -0,0 +1,10 @@
+// @flow weak
+
+import React from 'react';
+import Icon from 'material-ui/Icon';
+
+export default function AccentIcon() {
```

Comment:
These are _so_simple, I will combine them all.  For more complex variations, we can stick to multiple files.